edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/BrownClusterViewGenerator.java

```diff
@@ -107,6 +106,24 @@ private void loadClusters() throws Exception {
         if (gzip)
             stream = new GZIPInputStream(stream);
 
+        loadFromInputStream(stream);
+    }
+
+    public void loadFromDataStore() throws Exception {
+        Datastore dsNoCredentials = null;
+        try {
+            dsNoCredentials = new Datastore("http://smaug.cs.illinois.edu:8080");
```

Comment:
Possibly for future, I think we should add a `ResourcesConfigurator` to the core-utils to contain these details.